postgresql-client/recipes/install.rb

```diff
@@ -3,41 +3,41 @@
   action :remove
 end
 
-# Install Postgres 10 Client
-remote_file "#{Chef::Config[:file_cache_path]}/postgresql10-libs.rpm" do
-  source "https://jiffy-linux-packages.s3.amazonaws.com/postgresql10-libs-10.10-1PGDG.rhel6.x86_64.rpm"
+# Install Postgres 11 Client
+remote_file "#{Chef::Config[:file_cache_path]}/postgresql11-libs.rpm" do
+  source "https://jiffy-linux-packages.s3.amazonaws.com/postgresql11-libs-11.9-1PGDG.rhel6.x86_64.rpm"
   action :create
```

Comment:
There's actually no need to specify `action :create` because it's the default one.